eq-author-api/schema/tests/basicAnswer.test.js

```diff
@@ -398,4 +398,54 @@ describe("basic answer", () => {
       expect(page.id).toEqual(questionnaire.sections[0].pages[0].id);
     });
   });
+
+  describe("Versioning", () => {
```

Comment:
I think this test should be in `questionnaire.test.js`. It doesnt seem very specific to a `basicAnswer`.

Reply:
That makes sense, I've moved it across.